ui/main_window.py

```diff
@@ -0,0 +1,126 @@
+from ui.plot_window import PlotWindow
+from PyQt5.QtWidgets import * 
+from PyQt5.QtGui import * 
+from PyQt5.QtCore import * 
+import os
+
+DEFAULT_FILEPICKER_PATH = 'c:\\'
```

Comment:
```suggestion
DEFAULT_FILEPICKER_PATH = '~'
```

I think this will be used more on Linux so lets do this